module/app/app.go

```diff
@@ -553,13 +556,13 @@ func NewGravityApp(
 		minttypes.ModuleName,
 		crisistypes.ModuleName,
 		ibchost.ModuleName,
-		genutiltypes.ModuleName,
 		evidencetypes.ModuleName,
 		ibctransfertypes.ModuleName,
 		paramstypes.ModuleName,
 		upgradetypes.ModuleName,
 		vestingtypes.ModuleName,
 		gravitytypes.ModuleName,
+		genutiltypes.ModuleName,
```

Comment:
Genutil init genesis invoke `SetDelegateKeys` which would panic because the genesis params of the gravity module would not be populated in the previous order